### DISCUSSION TITLE
Google Docs Suite (Docs, Sheets, Slides, and maybe Forms)

### DISCUSSION THREAD

**Discussion by Fnige** [catppuccin/catppuccin#1844] [Port Requests]
**Are you available to help in the creation of this port? 😄 (optional)**
Maybe (Largely depends on difficulty)

**Additional context 📷 (optional)**
If you are willing to, if would be helpful to know if any theme exists at all for the google suite (Im pretty sure docs exist, but not the rest)

**Comment by nonetrix** [comment]
To my understanding Google docs uses ```<canvas>``` these days so might be a lot harder

**Reply by Oman395** [reply]
I believe that's just the editor window, which shouldn't be themed-- I'm going to look into using stylus for this

**Comment by ghost** [comment]
Any updates?

**Reply by isabelroses** [reply]
There is currently a WIP, see https://discord.com/channels/907385605422448742/1106785999750709339 for more, but if your not a discord user you can see the preview below:
![image](https://github.com/catppuccin/catppuccin/assets/71222764/4f40e7ad-c708-4cae-9714-085e681c9725)

**Comment by sgoudham** [comment]
Google Drive is being implemented in https://github.com/catppuccin/userstyles/pull/867

**Reply by isabelroses** [reply]
Google drive was merged a while ago, just thought I should leave an update,

**Comment by WalkQuackBack** [comment]
Reposting from Discord, a current functional iteration of this:
> More of a tech demo than an actually good theme, but layer these two userstyles on top of another.
> https://github.com/WalkQuackBack/google-suite-color-theme/raw/refs/heads/main/build.user.css
https://gist.github.com/WalkQuackBack/f3c200199a3aacae3d4ad43b437e7c3a/raw/b51e79916a2f260e9d43ee0d82eda8430570429f/ctp-docs.user.less
